core/src/Kokkos_Core.hpp

```diff
@@ -255,6 +255,7 @@ class ScopeGuard {
 
 #include <Kokkos_Crs.hpp>
 #include <Kokkos_WorkGraphPolicy.hpp>
+#include <impl/Kokkos_Combined_Reducer.hpp>
```

Comment:
Including it in `Kokkos_Parallel_Reduce.hpp` led to a circular dependency because `Kokkos::Sum` is used in `Kokkos_Combined_Reducer.hpp` and the default. The real answer is to finally break up `Kokkos_Parallel_Reduce.hpp` into smaller parts, but I didn't want to do too much all at once.

Reply:
Please add a comment to that effect